gui_qt/ab_experiment_report.py

```
@@ -0,0 +1,478 @@
+    if isinstance(chunk_count, int):
+        facts.append(f"采样块数：{chunk_count}")
+    if isinstance(variant_count, int):
+        facts.append(f"变体数：{variant_count}")
+    if isinstance(dry_run, bool):
+        facts.append(f"试跑模式：{'是' if dry_run else '否'}")
+    if isinstance(report_path, str) and report_path:
+        facts.append(f"报告：{report_path}")
+
+    findings: list[str] = []
+    if isinstance(report_path, str) and report_path and not os.path.isfile(report_path):
+        findings.append("命令已结束，但报告文件尚未找到。")
```

Ruff (0.15.20)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)